### DISCUSSION TITLE
Hide Window Decorations in kitty @ launch?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#6981] [Q&A]
Is it possible to hide window decorations _only_ within an OS window created via `kitty @ launch`?

Something like:

```
kitty @ launch --type=os-window -o hide_window_decorations=yes
```

`kitty` itself accepts the `-o, --override` command line option, but `kitty @ launch` does not.

**Comment by kovidgoyal** [comment]
No, hide window decorations is a global setting that applies to all OS
windows in a kitty instance.
